README.md

```diff
@@ -1,122 +1,129 @@
+* [MicroProfile Metrics](https://github.com/eclipse/microprofile-metrics)
+* [MicroProfile OpenAPI](https://github.com/eclipse/microprofile-open-api)
+* [MicroProfile OpenTracing](https://github.com/eclipse/microprofile-opentracing)
+
+More tests could be added, specifically covering the MicroProfile 
+[REST Client](https://github.com/eclipse/microprofile-rest-client), 
+[Reactive Messaging](https://github.com/eclipse/microprofile-reactive-messaging) and
+[Reactive Streams Operators](https://github.com/eclipse/microprofile-reactive-messaging) (just tech preview at the 
+moment) specs.
 
 ## Requirements for the testsuite
- - JDK 8+
- - Docker
+- JDK 8+
+- Docker
  
- Maven is not a hard requirement as `mvnw` is delivered with this repository.
-
- Podman with docker command bridge can be used as a Docker alternative or you can use a system property `-Ddocker.command=podman`.
+Maven is not a hard requirement as `mvnw` is delivered with this repository.
+
+Podman can be aliased to be used as a Docker alternative - i.e. `alias docker=podman`, or you can pass the 
+`-Ddocker.command=podman` system property to the test suite when executing Maven.
+
+## Modules
+
+### MicroProfile specs testing modules
+- [microprofile-config](./microprofile-config) - contains MicroProfile Config deployments and tests
+- [microprofile-fault-tolerance](./microprofile-fault-tolerance) - contains MicroProfile Config deployments and tests
+- [microprofile-health](./microprofile-health) - contains MicroProfile Config deployments and tests
+- [microprofile-jwt](./microprofile-jwt) - contains MicroProfile Config deployments and tests
+- [microprofile-metrics](./microprofile-metrics) - contains MicroProfile Config deployments and tests
+- [microprofile-open-api](./microprofile-open-api) - contains MicroProfile Config deployments and tests
+- [microprofile-opentracing](./microprofile-opentracing) - contains MicroProfile Config deployments and tests
+
+### Tooling modules
+- [tooling-cpu-load](./tooling-cpu-load) - utilities to support performance testing as the 
+  [microprofile-fault-tolerance  `CpuLoadTest`](./microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/CpuLoadTest.java)
+- [tooling-docker](./tooling-docker) - utilities needed to run and manage `docker` as a part a given test scenario,
+  e.g.: the 
+  [microprofile-fault-tolerance `DatabaseCrashTest`](./microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/DatabaseCrashTest.java)
+- [tooling-mp-jwt-auth-tool](./tooling-mp-jwt-auth-tool) - utilities required by security operations, e.g.: keys' 
+  generation utilities, which are used by the [microprofile-jwt](./microprofile-jwt) module
+- [tooling-server-configuration](./tooling-server-configuration) - server configuration utilities, see the module 
+[README](./tooling-server-configuration/README.md)
 
 ## Run the testsuite
+Just execute:
```

Comment:
Hmm, this looks like both commands are necessary, can you please clarify it in the readme?

Reply:
Yeah, sure, it can be improved.